cmd/atlas/atlas.go

```diff
@@ -36,6 +37,10 @@ func execute(rootCmd *cobra.Command) {
 
 To learn more, see our documentation: https://www.mongodb.com/docs/atlas/cli/stable/connect-atlas-cli/`
 	if cmd, err := rootCmd.ExecuteContextC(ctx); err != nil {
+		errMsg := commonerrors.CheckHTTPErrors(err)
+		if errMsg != nil {
+			fmt.Println(errMsg)
+		}
```

Comment:
shouldn't we print this to stderr?

Reply:
I think this prints it twice because cobra prints it

Reply:
If we are moving the error check here we should remove each individual usage of it and move it here